README.md

```diff
@@ -50,13 +50,14 @@ The Testkube Agent (this repo) is **100% Open-Source** and can be deployed stand
 - **Works with your tools**: Integrate with existing tools and infrastructure using [Webhooks](https://docs.testkube.io/articles/webhooks) and the [Testkube REST API](https://docs.testkube.io/openapi/overview) - see [Integration Examples](https://docs.testkube.io/articles/integrations).
 - **Enterprise Ready**: SSO/SCIM, RBAC, Teams, Resource-Groups, Audit-logs, etc. - [Read More](https://docs.testkube.io/articles/administration-overview).
 
-### Getting Started
+### Getting Started with Enterprise
```

Comment:
suggest to have a "Getting Started with Open Source" section that links accordingly, and a separate "Getting Started with the Commercial Control Plane" section ?